x-pack/plugins/code/public/components/main/side_tabs.tsx

```diff
@@ -22,6 +25,7 @@ enum Tabs {
 interface Props extends RouteComponentProps<MainRouteParams> {
   loadingFileTree: boolean;
   loadingStructureTree: boolean;
+  hasStucture: boolean;
```

Comment:
this `hasStructure` is newly introduced in state/props, but I don't see where it's applied. Did you forget to push other changes in your local machine?